### PR TITLE
Conda.cleanall()

### PR DESCRIPTION
**Pull request by rofinn**
Added a Conda.cleanall function for removing unused packages and artifacts.

### CODE REVIEW

File: src/Conda.jl [hunk 1]
```diff
@@ -289,4 +289,25 @@ function rm_channel(channel::AbstractString, env::Environment=ROOTENV)
     runconda(`config --remove channels $channel --force`, env)
 end
 
+"Cleanup all unused packages and caches"
+function cleanall(; debug=false)
+    clean(; debug=debug, index=true, locks=true, tarballs=true, packages=true, sources=true)
+end
+
+function clean(;
```
Comment (stevengj) [inline src/Conda.jl]: Needs a docstring.  What does this do with the default arguments?

Comment (rofinn) [reply src/Conda.jl]: Nothing. I suppose we could throw a warning in that condition.

Comment (stevengj) [reply src/Conda.jl]: If `clean()` does nothing, why not make it do something useful: just make it do`cleanall()` then, and get rid of the `cleanall` function?

Comment (rofinn) [reply src/Conda.jl]: So have the default kwargs be `true`? That seemed like a dangerous default, but I'm personally always doing `cleanall` anyways.

Comment (stevengj) [reply src/Conda.jl]: Yes.

**Pull request merged**